profiles/features/musl/package.mask

```diff
@@ -57,6 +57,11 @@ net-print/pnm2ppa
 net-wireless/neard
 sys-apps/noexec
 
+# Giuseppe Foti <foti.giuseppe@gmail.com> (2023-05-19)
+# New packages that require blocked packages in the previous block.
```

Comment:
I don't see a dependency of notus-scanner "in the previous block"? But maybe I am missing something.
```suggestion
# New packages that require packages masked by the previous block.
```